package.json

```
@@ -26,11 +26,11 @@
     "coffee-script": "~1.7.0",
     "express": "4.17.1",
     "lodash": "^4.17.13",
-    "logger-sharelatex": "^1.7.0",
-    "metrics-sharelatex": "^2.5.1",
-    "mongojs": "^2.6.0",
+    "logger-sharelatex": "^1.9.0",
```

In the logger module as well:
```suggestion
    "logger-sharelatex": "^1.9.1",
```